platform/api/do/api.go

```diff
@@ -199,6 +210,43 @@ func (a *API) DeleteDroplet(ctx context.Context, dropletID int) error {
 	return nil
 }
 
+func (a *API) GetUserImage(ctx context.Context, imageName string) (*godo.Image, error) {
+	var ret *godo.Image
+	page := godo.ListOptions{
+		PerPage: 200,
+	}
+	for {
+		images, _, err := a.c.Images.ListUser(ctx, &page)
```

Comment:
Does this call return an empty list with no error in the case that the page you're requesting doesn't exist? Otherwise is it possible to fall into the error below if the user has exactly 200 images?

Reply:
Good question; will check.

Reply:
Empty list. It also turns out that the pages are 1-indexed and not 0-indexed; fixed.